docs/migrations/v2-to-v3.md

```diff
@@ -42,7 +59,12 @@ Please review the [mock](../../testing/mock/ibc_module.go) and [transfer](../../
 
 ## Relayers
 
-- No relevant changes were made in this release.
+`AppVersion` gRPC has been removed.
+The `version` string in `MsgChanOpenTry` has been deprecated and will be ignored by core IBC. 
+Relayers no longer need to determine the version to use on the `ChanOpenTry` step.
+IBC applications will determine the correct version using the counterparty version. 
+
+Relayers may need to ensure the counterparty version passed in on `ChanOpenAck` matches the version set by the `ChanOpenTry` step.
```

Comment:
Why? We'd just grab the counterparty version from TRY events. If the value is wrong, the proof will fail.

I would just remove this line, since on reading it seems like relayers need to be doing something different. whereas they just do the same relaying from events + state queries